### ISSUE TITLE
MACHC-WAE9LP Support (i7-10510U)

### ISSUE THREAD

**Issue by Rino53** [RepoWeaver/MateBook-X-Pro-2020-OpenCore#11]
Does this build support model MateBook X Pro 2020 i7-10510U 16GB+1TB (MACHC-WAE9LP) ?
Its not clear from readme, and i dont know where else i can ask such a question.

**Comment by RepoWeaver** [comment]
Yes it should as long as your SSD is supported then it should be

 

From: Rinat Salakhov ***@***.***> 
Sent: 3 October 2022 6:06 PM
To: RepoWeaver/MateBook-X-Pro-2020-OpenCore ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [RepoWeaver/MateBook-X-Pro-2020-OpenCore] MACHC-WAE9LP Support (i7-10510U) (Issue #11)

 

Does this build support model MateBook X Pro 2020 i7-10510U 16GB+1TB (MACHC-WAE9LP) ?
Its not clear from readme, and i dont know where else i can ask such a question.

**Comment by Rino53** [comment]
SSD in this model is following: 1TB Samsung MZVLB1T0HBLR-00000
How to know if its supported or not ?

**Comment by RepoWeaver** [comment]
I’m running a WDC SSD but I think Samsungs aren’t supported but you can try the install as I haven’t tested on your sdd Kind Regards,Chigozie UzomaOn 6 Oct 2022, at 9:06 am, Rinat Salakhov ***@***.***> wrote:﻿
SSD in this model is following: 1TB Samsung MZVLB1T0HBLR-00000
How to know if its supported or not ?

**Comment by RepoWeaver** [comment]
It might be supported but you’ll have to try this before hand
